docs/manual/fault_domains.md

```diff
@@ -188,7 +188,25 @@ spec:
 
 Once all three `FoundationDBCluster` resources are marked as reconciled the FoundationDB cluster is up and running.
 You can run this configuration in the same namespace, different namespaces or even across multiple different Kubernetes clusters.
-Operations across the different `FoundationDBCluster` resources are [coordinated](#coordinating-global-operations). 
+Operations across the different `FoundationDBCluster` resources are [coordinated](#coordinating-global-operations).
+
+## Migrating an existing cluster to Three-Data-Hall Replication
+
+It is possible to gracefully migrate a cluster in single, double or triple replication to Three-Data-Hall replication by following these steps:
+
+1. create 3 exact clones of the original k8s FoundationDB cluster object (thus still with same replication) and change these fields: `metadata.name`,`spec.processGroupIDPrefix`, `spec.dataHall`, `spec.dataCenter`
+2. make sure that `skip: true` is set on their YAML definition so that operator will not attempt to configure them
+3. make sure that `seedConnectionString:` is set to to the current connection string of the original cluster
+4. run `kubectl create` for each of them
+5. set the configured state and connection string in their status subresource by using: `kubectl patch fdb my-new-cluster-a --type=merge --subresource status --patch "status: {configured: true, connectionString: \"...\" }"`; use the original cluster connection string here
+6. set `skip: false` on each of the 3 new clusters, and then wait for reconciliation to finish
+7. start a lengthy exclude procedure that will exclude all the processes of the original cluster; suggested order: `log`, `storage`, `coordinator`, `stateless`
```

Comment:
Before doing that shouldn't we set `skip: true` on the original cluster?

Reply:
I experimentally found that the reconciliation at previous step 6 was necessary to mitigate 1031 timeouts; I think this is because waiting for reconciliation also waits for data movement to finish (operator cannot reconcile if data state is not healthy for a certain amount of time).